target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -115,6 +115,23 @@ define Device/netgear_rax120v2
 endef
 TARGET_DEVICES += netgear_rax120v2
 
+define Device/netgear_wax620
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	ARTIFACTS := web-ui-factory.fit
+	DEVICE_VENDOR := Netgear
+	DEVICE_MODEL := WAX620
+	DEVICE_DTS_CONFIG := config@hk07
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	SOC := ipq8072
+	ARTIFACT/web-ui-factory.fit := append-image initramfs-uImage.itb | \
```

Comment:
Is it being used for installation or?

Reply:
The web-ui image is not being used. Only squashfs-factory.ubi, squashfs-sysupgrade.bin and initramfs-uImage.itb. 

Do I delete the ARTIFACT line 128?
```suggestion
```

Reply:
Yes, remove that line as well as the one listing the artifacts.

Reply:
New commit pushed.